vignettes/test_data_guidance.Rmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Test Data Guidance"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Test Data Guidance}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+[`admiraldata`](https://github.com/pharmaverse/admiraldata) provides a one-stop-shop for test data in the [`admiral`](https://pharmaverse.github.io/admiral/cran-release/) family of packages. This includes datasets that are therapeutic area (TA)-agnostic (`DM`, `VS`, `EG`, etc.) as well TA-specific ones (`RS`, `TR`, `OE`, etc.).
+
+# Data Sources
+
+Some of the test datasets has been sourced from the [CDISC pilot project](https://github.com/cdisc-org/sdtm-adam-pilot-project), while other datasets have been constructed ad-hoc by the admiral team. Please check the [Github repository](https://github.com/pharmaverse/admiral.test/tree/main/data) for detailed information regarding the source of specific datasets.
+
+# Naming Conventions {#Name}
+
+-   Datasets/programs that are TA-agnostic are prefixed with `admiral_` (e.g. `admiral_dm`).
+-   Datasets/programs that are TA-specific are prefixed with the `admiral` extension package from which they derive (e.g. `admiralonco_rs`, `admiralophtha_oe`).
+-   Consistent name between datasets and programs.
+
+**Note**: *If a domain is used by multiple TAs, [`admiraldata`](https://github.com/pharmaverse/admiraldata) may provide multiple versions of the corresponding test dataset. For instance, the package contains `admiral_ex` and `admiralophtha_ex` as the latter contains ophthalmology-specific variables such as `EXLAT` and `EXLOC`, and `EXROUTE` is exchanged for a plausible ophthalmology value.*
+
+# How To Update
+
+Firstly, make a GitHub issue in this repo with the planned updates and tag `@pharmaverse/admiral` so that one of the development core team can sanity check the request. Then there are two main ways to extend the test data: either by adding new datasets or extending existing datasets with new records/variables. Whichever method you choose, it is worth noting the following:
```

Comment:
change "this repo" to {pharmaversesdtm}

Reply:
updated here: https://github.com/pharmaverse/pharmaversesdtm/pull/42